### DISCUSSION TITLE
error parameter select dengan dplyr

### DISCUSSION THREAD

**Discussion by rastitiratnasari** [teamalgoritma/community#848] [Q&A] [answered]
Kenapa ya selalu ada error ketika menggunakan  parameter select.
selalu muncul error :
Error in select(., -c(X, EASE.MENT, APARTMENT.NUMBER)) : 
  unused argument (-c(X, EASE.MENT, APARTMENT.NUMBER)).

sdh dilakukan run all above, dan sdh ditambahkan library(dplyr)

**Comment by IrfanChairurrachman** [comment, marked as answer]
Dear @rastitiratnasari ,

Boleh disertakan kode awal ketika melakukan select bu?
Untuk melakukan select di dplyr kita tidak membutuhkan c() untuk drop kolom yang tidak diinginkan. Untuk drop kolom di dpylr bisa langsung dengan
```
data %>% select(-Kolom1, -Kolom2, -Kolom3)
```
atau dalam kasus Ibu akan seperti:
```
data %>% select(-X, -EASE.MENT, -APARTMENT.NUMBER)
```

Demikian jawaban yang dapat kami sampaikan. Semoga membantu! Jika ternyata jawaban ini menyelesaikan permasalahan Anda, silakan mark as anwer.
